Samples/iOS-Swift/iOS-SwiftUITests/SentryEndToEndTests.swift

```diff
@@ -0,0 +1,22 @@
+import XCTest
+
+#if os(iOS)
+class SentryEndToEndTests: XCTestCase {
```

Comment:
This doesn't do anything yet. So I guess you want to add something here right?

Reply:
Old file, removed in favor of a different one that actually contains the test logic in #1834